dags/run_dbt_cleaning_gpt.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.models import Connection
+from airflow.operators.bash import BashOperator
+from airflow.operators.docker_operator import DockerOperator
+from docker.types import Mount
+from airflow.operators.python import PythonOperator
+import logging
+
+def print_debut():
+    logging.info(f"sa path variable : {os.getenv('sapath')}")
+
+with DAG(
+    'run_dbt_cleaning',
+    description='Check if dataset exist and create if not',
```

Comment:
Proposition: Transform raw data

Reply:
changed it to transform_raw_to_silver for explicitly